### PR TITLE
feat(i18nService): display key when label is not found

### PR DESCRIPTION
**Pull request by hututudor**
https://etvas.atlassian.net/browse/ETV-10020

### CODE REVIEW

File: src/I18nService.js [hunk 1]
```diff
@@ -36,39 +36,31 @@ export class I18nService extends EventEmitter {
...
+        markedRule =>
+          dictionary?.[`${label}.${markedRule.suffix}`] &&
+          markedRule.condition(marked)
+      )
+
+      if (rule) {
+        return this.replacePlaceholders(
+          dictionary[`${label}.${rule.suffix}`],
+          args
+        )
```
Comment (raducastravet) [inline src/I18nService.js]: I'm not sure this is very readable. Could you do something about it please? Maybe extract something in another function or rename some variables.

File: src/I18nService.js [hunk 2]
```diff
@@ -36,39 +36,31 @@ export class I18nService extends EventEmitter {
...
-        return this.replacePlaceholders(dictionary[`${label}.${found}`], args)
+    const marked = args?.[mark]
+    if (mark && !isNaN(marked)) {
+      const rule = markedRules.find(
+        markedRule =>
+          dictionary?.[`${label}.${markedRule.suffix}`] &&
+          markedRule.condition(marked)
+      )
+
+      if (rule) {
```
Comment (ohnu) [inline src/I18nService.js]: what is a `rule`, `mark`, `markedRule` ? let's use some well known terms instead like
`key`, `value`, `translation` or something

Comment (hututudor) [reply src/I18nService.js]: This is out of scope for this PR ... if we want to refactor these terms (and I'm agreeing we should) we need to make a separate task for this

Comment (ohnu) [reply src/I18nService.js]: this is the perfect time to rename the code you touched .. not code from other files .. not code from other functions .. this code.. this exact 20 lines or so 

In order to change it you must understand it and be clear to you what is doing ... in order for me to review it I also need to understand it

Comment (hututudor) [reply src/I18nService.js]: If I have to rename `mark` for example, I have to also update it in the portals (since it is public property)

Comment (ohnu) [reply src/I18nService.js]: that is a good reason for that one .. but let's do the local ones only
and remember that we can also rename the inputs locally

File: src/I18nService.js (hunk 1 above)
Comment (hututudor) [reply src/I18nService.js]: ab0036f3db4ff2353fa5a23f7286f2df41bfd22d

File: src/I18nService.js (hunk 2 above)
Comment (hututudor) [reply src/I18nService.js]: https://github.com/etvascom/i18n/pull/20/commits/6df4e533e81696654cf90862f2e2c0210a84c893

File: src/I18nService.js [hunk 3]
```diff
@@ -49,6 +50,13 @@ export class I18nService extends EventEmitter {
       return this.replacePlaceholders(dictionary[key], args)
     }
 
+    if (language !== this.options.defaultLanguage && !this.fallbackToKey) {
+      console.warn(
```
Comment (ohnu) [inline src/I18nService.js:54]: vrem acest warning in consola la fiecare cheie tradusa ? daca traduci un ecran intreg o sa fie plina consola

Comment (hututudor) [reply src/I18nService.js:54]: Da ... vrem sa stim cand o cheie lipseste (asa era bahaviourul inainte)

File: src/I18nService.js (hunk 2 above)
Comment (ohnu) [reply src/I18nService.js]: `const marked = args?.[pluralBy]`

**Review by ohnu — APPROVED** [review]

File: src/I18nService.js (hunk 2 above)
Comment (hututudor) [reply src/I18nService.js]: 1a52a1211b3bdf7de19cf82eab109b5fbbdf059f

**Review by raducastravet — APPROVED** [review]

**Pull request merged**